terraform/azure_server/files/install_wsl.ps1

```diff
@@ -0,0 +1,5 @@
+Invoke-WebRequest -Uri "https://aka.ms/wsl-ubuntu-1804" -OutFile $env:TEMP\wsl.zip -UseBasicParsing
+Expand-Archive $env:TEMP\wsl.zip "$env:windir\System32\wsl" -Force
+
+Add-MpPreference -ExclusionProcess "$env:windir\System32\wsl\ubuntu1804.exe"
+& "$env:windir\System32\wsl\ubuntu1804.exe" install --root
```

Comment:
this may not succeed at first try by the scripts since you may need elevated permissions, but added a note to the docs about this. Regardless, this script gets `C:\Windows\System32\wsl` populated so that you just need to run one command to set up the install manually.